File: frontend/src/constants/legal_object.ts
import { Dayjs } from "dayjs";

export type legal_object = {
  
  id: number;
  description: string;
  address: string;
  geojson: string;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
};

File: backend/Domain/Entities/legal_object.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class legal_object
    {
        public int id { get; set; }
		public string description { get; set; }
		public string address { get; set; }
		public string geojson { get; set; }
		public DateTime? created_at { get; set; }
		public DateTime? updated_at { get; set; }
		public int? created_by { get; set; }
		public int? updated_by { get; set; }
		
    }
}